scrapy/exporters.py

```diff
@@ -96,9 +100,13 @@ class JsonItemExporter(BaseItemExporter):
     def __init__(self, file, **kwargs):
         self._configure(kwargs, dont_fail=True)
         self.file = file
-        self.encoder = ScrapyJSONEncoder(**kwargs)
+        self.encoder = ScrapyJSONEncoder(ensure_ascii=self.ensure_ascii, **kwargs)
         self.first_item = True
 
+    def _configure(self, options, dont_fail=False):
+        self.ensure_ascii = options.pop('ensure_ascii', True)
```

Comment:
I added check for ensure_ascii from the options, so you would not have to subclass the exporter, just pass the option to it.

Setting would probably be a better option though. Kk, I'll add that one too.

As for the encoding parameter: JSON encode accepts separate ensure_ascii and encoding parameters. Would you like them to be separate like that? Or turn of ensure_ascii automatically if the encoding is specified?